docs/ops/reduction/ReduceL2_4.md

```diff
@@ -0,0 +1,65 @@
+## ReduceL2 <a name="ReduceL2"></a>
+
+**Versioned name**: *ReduceL2-4*
+
+**Category**: *Reduction*
+
+**Short description**: *ReduceL2* operation performs reduction with finding the L2 norm of the 1st input tensor in slices specified by the 2nd input.
+
+**Attributes**
+
+* *keep_dims*
+
+  * **Description**: If set to `True` it holds axes that are used for reduction. For each such axis, output dimension is equal to 1.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: Input tensor x of any data type that has defined L2 norm operation. **Required.**
+
+* **2**: Scalar or 1D tensor with axis indices for the 1st input along which reduction is performed. **Required.**
+
+**Outputs**
+
+* **1**: Tensor of the same type as the 1st input tensor and `shape[i] = shapeOf(input1)[i]` for all `i` that is not in the list of axes from the 2nd input. For dimensions from the 2nd input tensor, `shape[i] == 1` if `keep_dims == True`, or `i`-th dimension is removed from the output otherwise.
+
+**Detailed Description**
+
+Each element in the output is the result of reduction with finding a L2 norm operation along dimensions specified by the 2nd input:
+
+    output[i0, i1, ..., iN] = L2[j0,..., jN](x[j0, ..., jN]**2))
```

Comment:
I don't understand the formula. What is `x`?

Reply:
We use same formula style in all reduce operations, should we need to update all of them?

Reply:
Yes. Also, it looks like a bug in formulas for other reduce operations. For example, in ReduceMin the formula is the following:

`output[i0, i1, ..., iN] = min[j0,..., jN](x[j0, ..., jN]**2))`

I assume that `x**2` means the `x` in the power of 2. But it is not needed to raise the number `x` in the power of two.